AEPMessaging/Sources/Messaging.swift

```diff
@@ -90,8 +83,18 @@ public class Messaging: NSObject, Extension {
             Log.debug(label: MessagingConstants.LOG_TAG, "Event processing is paused, waiting for valid xdm shared state from edge identity - '\(event.id.uuidString)'.")
             return false
         }
-
-        return configurationSharedState.status == .set && edgeIdentitySharedState.status == .set
+        
+        if configurationSharedState.status != .set || edgeIdentitySharedState.status != .set {
```

Comment:
can we add these checks in the previous guard statements for respective extensions.. so that it is being logged as well
```
  guard let configurationSharedState = getSharedState(extensionName: MessagingConstants.SharedState.Configuration.NAME, event: event), configurationSharedState.status == .set else 
  {
    // log it
   }
```